<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 243, 70);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="numeros">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorMediana">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorMenor">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultado">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>110</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;
  </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorDesviacion">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorModa">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorMedia">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2ecc71;
color: white;
border: 2px solid black;
border-radius: 12px;
padding: 10px 20px;
font-size: 14px;
font-weight: bold;
text-align: center;
transition: background-color 0.3s, transform 0.2s;</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrarb">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e74c3c;
color: white;
border: 2px solid black;
border-radius: 8px;
padding: 8px 16px;
font-size: 14px;
font-weight: bold;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Borrar dataset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcularb">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2ecc71;
color: white;
border: 2px solid black;
border-radius: 12px;
padding: 10px 20px;
font-size: 14px;
font-weight: bold;
text-align: center;
transition: background-color 0.3s, transform 0.2s;</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorVarianza">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);
</string>
    </property>
    <property name="text">
     <string>Valor Menor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Valor Mayor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValorMayor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
border: 2px solid #3498db;
border-radius: 10px;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardarb">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3498db;
color: white;
border: 2px solid black;
border-radius: 8px;
padding: 8px 12px;
font-size: 14px;
font-weight: bold;
text-align: center;
transition: background-color 0.3s, transform 0.2s;</string>
    </property>
    <property name="text">
     <string>Guardar Resultados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cargarrb">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3498db;
color: white;
border: 2px solid black;
border-radius: 8px;
padding: 8px 12px;
font-size: 14px;
font-weight: bold;
text-align: center;
transition: background-color 0.3s, transform 0.2s;</string>
    </property>
    <property name="text">
     <string>Cargar resultados anteriores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrarub">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2ecc71;
color: white;
border: 2px solid black;
border-radius: 12px;
padding: 10px 20px;
font-size: 14px;
font-weight: bold;
text-align: center;
transition: background-color 0.3s, transform 0.2s;</string>
    </property>
    <property name="text">
     <string>borrar ultimo numero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/logos/logo equipo 3.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/logos/logo equipo 3.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Media</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Mediana</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Moda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Desviacion Estandar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Times New Roman&quot;;
color: rgb(158, 74, 161);</string>
    </property>
    <property name="text">
     <string>Varianza</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Archivos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
